lib/sacastats_web/views/character_view.ex

```diff
@@ -98,23 +138,81 @@ defmodule SacaStatsWeb.CharacterView do
   end
 
   defp build_event_log_item(assigns, %PlayerFacilityCapture{} = cap, %Session{} = session, _c_map) do
+    facility = SacaStats.facilities()[cap.facility_id]
+
+    facility_type_text =
+      if facility["facility_type"] in ["Small Outpost", "Large Outpost"] do
+        ""
+      else
+        facility["facility_type"]
+      end
+
+    # can't do this right now, need outfit ID from FacilityControl events (the one provided here is just the player's
+    # current outfit :/)
+    outfit_captured_text = ""
+    # if cap.outfit_id == session.outfit.outfit_id do
+    #   "for #{session.outfit.name}!"
+    # else
+    #   ""
+    # end
+
     ~H"""
     <li>
-      <%= session.name %> captured <%= cap.facility_id %> (new outfit owner: <%= cap.outfit_id %>) -
+      <%= format_character_link(session.name) %> captured
+      <%= "#{facility["facility_name"] || "a facility"} #{facility_type_text}" %>
+      <%= outfit_captured_text %> -
       <%= SacaStatsWeb.CharacterView.prettify_timestamp(assigns, cap.timestamp) %>
     </li>
     """
   end
 
   defp build_event_log_item(assigns, %PlayerFacilityDefend{} = def, %Session{} = session, _) do
+    facility = SacaStats.facilities()[def.facility_id]
+
+    facility_type_text =
+      if facility["facility_type"] in ["Small Outpost", "Large Outpost"] do
+        ""
+      else
+        facility["facility_type"]
+      end
+
+    outfit_captured_text = ""
+    # if def.outfit_id == session.outfit.outfit_id do
```

Comment:
Was this commented out on purpose?

Reply:
Yeah, for the same reason as L150 above (once we start listening to FacilityControl events and do proper member-outfit contributions/tracking, I'll probably move this logic there).